enclave/src/main/java/org/hyperledger/besu/enclave/Enclave.java

```diff
@@ -54,42 +56,72 @@ public boolean upCheck() {
     }
   }
 
-  public SendResponse send(final SendRequest content) {
+  public SendResponse sendLegacy(
+      final String payload, final String privateFrom, final List<String> privateFor) {
+    final SendRequestLegacy sendRequestLegacy =
+        new SendRequestLegacy(payload, privateFrom, privateFor);
     return post(
         JSON,
-        content,
+        sendRequestLegacy,
         "/send",
         (statusCode, body) -> handleJsonResponse(statusCode, body, SendResponse.class));
   }
 
-  public ReceiveResponse receive(final ReceiveRequest content) {
+  public SendResponse sendBesu(
+      final String payload, final String privateFrom, final String privacyGroupId) {
+    final SendRequestBesu sendRequestBesu =
+        new SendRequestBesu(payload, privateFrom, privacyGroupId);
+    return post(
+        JSON,
+        sendRequestBesu,
+        "/send",
+        (statusCode, body) -> handleJsonResponse(statusCode, body, SendResponse.class));
+  }
+
+  public ReceiveResponse receive(final String enclaveKey) {
+    final ReceiveRequest receiveRequest = new ReceiveRequest(enclaveKey);
+    return post(
+        ORION,
+        receiveRequest,
+        "/receive",
+        (statusCode, body) -> handleJsonResponse(statusCode, body, ReceiveResponse.class));
+  }
+
+  public ReceiveResponse receive(final String enclaveKey, final String to) {
+    final ReceiveRequest receiveRequest = new ReceiveRequest(enclaveKey, to);
     return post(
         ORION,
-        content,
+        receiveRequest,
         "/receive",
         (statusCode, body) -> handleJsonResponse(statusCode, body, ReceiveResponse.class));
   }
 
-  public PrivacyGroup createPrivacyGroup(final CreatePrivacyGroupRequest content) {
+  public PrivacyGroup createPrivacyGroup(
+      final String[] addresses, final String from, final String name, final String description) {
+    final CreatePrivacyGroupRequest createPrivacyGroupRequest =
+        new CreatePrivacyGroupRequest(addresses, from, name, description);
     return post(
         JSON,
-        content,
+        createPrivacyGroupRequest,
         "/createPrivacyGroup",
         (statusCode, body) -> handleJsonResponse(statusCode, body, PrivacyGroup.class));
   }
 
-  public String deletePrivacyGroup(final DeletePrivacyGroupRequest content) {
+  public String deletePrivacyGroup(final String privacyGroupId, final String from) {
+    DeletePrivacyGroupRequest deletePrivacyGroupRequest =
+        new DeletePrivacyGroupRequest(privacyGroupId, from);
     return post(
         JSON,
-        content,
+        deletePrivacyGroupRequest,
         "/deletePrivacyGroup",
         (statusCode, body) -> handleJsonResponse(statusCode, body, String.class));
   }
 
-  public PrivacyGroup[] findPrivacyGroup(final FindPrivacyGroupRequest content) {
+  public PrivacyGroup[] findPrivacyGroup(final String[] addresses) {
+    final FindPrivacyGroupRequest findPrivacyGroupRequest = new FindPrivacyGroupRequest(addresses);
```

Comment:
nit: here and elsewhere you can probably name the variable just request or content the extra naming isn't adding value in this context